installer/helm/chart/volcano/.helmignore

```diff
@@ -0,0 +1,2 @@
+# match crd
+crd/v1beta1/
```

Comment:
Why ignore this?

Reply:
![image](https://user-images.githubusercontent.com/52434045/181459374-8cecf110-c932-495d-a7eb-d8f8336c0bda.png)

The content of the chart is too large and eventually the Secret "sh.helm.release.v1.integration.v1" created is invalid. helm cannot deploy volcano successfully, ci cannot pass.

Reply:
@Thor-wl @shinytang6 @william-wang 
Is it acceptable to ignore the v1beta1 version of the crd?